src/View3d.ts

```diff
@@ -831,19 +831,24 @@ export class View3d {
    * Switch between single pass ray-marched volume rendering and progressive path traced rendering.
    * @param {number} mode 0 for single pass ray march, 1 for progressive path trace
    */
-  setVolumeRenderMode(mode: number): void {
+  setVolumeRenderMode(mode: RenderMode.PATHTRACE | RenderMode.RAYMARCH): void {
```

Comment:
Slightly odd to just expose these two options at this level... makes me think that, if other orthographic viewing angles get limited to a single slice, we could combine camera mode and render mode into a single concept (volumetric, pathtrace, slice x/y/z)

Reply:
Sadly, the enum is overloaded here. `setVolumeRenderMode` would never have any other values than these - it's only setting how the 3d volumetric rendering mode should behave.  The max-projection flag is also slightly confusingly blended in here.  It can be improved but at least this code change clarifies the contract and intention a little better.

If anything, I think this should be broken out into a separate enum that says what algorithm/shader to use in 3d mode. So you'd have an enum for "slice or volumetric" and if volumetric, there's a different enum for "raymarch or pathtrace".. something like that.  And then raymarch has a flag for whether it's doing max projection or not... kinda sorta maybe